modules/tinymce/src/plugins/link/test/ts/browser/UrlProtocolTest.ts

```diff
@@ -84,6 +84,9 @@ UnitTest.asynctest('browser.tinymce.plugins.link.UrlProtocolTest', (success, fai
       ]),
       Log.stepsAsStep('TBA', 'Test regex for email link', [
         testProtocolConfirm('no-reply@example.com', 'mailto:')
+      ]),
+      Log.stepsAsStep('TBA', 'Test regex for path with @', [
```

Comment:
Use the JIRA number instead of TBA here please.

Reply:
Sorry @trevorjay, I should have been clearer on this. It should be `TINY-5941` instead of just `5941`.

Reply:
No worries. I should have thought of that myself.